### ISSUE TITLE
Big count for "else".

### ISSUE THREAD

**Issue by bots-business** [Untoldhacker-Dev/friendly-octo-fortnight#63]
Your _withdraw.js:
```

if (!user.wallet) {
  Bot.sendMessage("_❌ Wallet Not set_")
} else if (!minimum_withdraw) {
...
}else
if (user.balance < minimum_withdraw) {
....
} else {
....
}
```

use return. Do not use "else". It is really not good with big body for case.
Also you can use function

**Comment by bots-business** [comment]
Simular for withdrawRequest.js

```
if (isNaN(message)) {
  isInvalid()
} else if (parseFloat(message) < Math.round(minimum_withdraw)) {
  needMoreAmount()
} else if (parseFloat(message) > user.balance) {
  needLessAmount()
} else sendWithdrawRequest()
```

**Comment by bots-business** [comment]
Please assign tag "need checking" after fixing for this issue

**Comment by Untoldhacker-Dev** [comment]
Fixed: #66
